lib/pharos/kube.rb

```diff
@@ -212,10 +228,9 @@ def self.delete_resource(host, resource)
     # @param path [String]
     # @return [Kubeclient::Resource]
     def self.parse_resource_file(path, vars = {})
-      yaml = File.read(File.realpath(File.join(__dir__, 'resources', path)))
-      parsed_yaml = Pharos::Erb.new(yaml).render(vars)
-
-      Kubeclient::Resource.new(YAML.safe_load(parsed_yaml))
+      Kubeclient::Resource.new(
+        Pharos::YamlFile.new(File.realpath(resource_path(path))).load(vars)
```

Comment:
The arg be something like `filename` instead of `path`, if `resource_files` => `Dir.glob` only returns the base filename.

Better yet, have `resource_files` just return the full paths.

Reply:
Hmm?

```
irb(main):005:0> Dir.glob('/tmp/*')
=> ["/tmp/12b91f1745676f37ea3b81e48869fd3c-{87A94AB0-E370-4cde-98D3-ACC110C5967D}", "/tmp/com.apple.launchd.SjQyHUEOBP", "/tmp/com.apple.launchd.zV0kvlYB3q", "/tmp/jorma"]
```

Reply:
Then why the `resource_path(path)`?

I was looking at the examples in the docs too literally...

Reply:
I think maybe mostly because of ./spec/pharos/kube_spec.rb:4

```ruby
      it 'returns resource' do
        resource = described_class.parse_resource_file('ingress-nginx/03-role.yml')
        expect(resource.metadata.name).to eq('nginx-ingress-role')
      end
```

which called it using a relative path.